insert permissions with backend privileges (RFC #4120) (#4224)
hasura/graphql-engine: docs/graphql/manual/api-reference/schema-metadata-api/permission.rst

     - false
     - :ref:`PGColumn` array (or) ``'*'``
     - Can insert into only these columns (or all when ``'*'`` is specified)
   * - admin_only
     - false
     - Boolean

>      - When set to ``true`` the mutation is accessible only if role header (``x-hasura-role``)

Reviewer: ```suggestion
     - When set to ``true`` the mutation is accessible only if the role header (``x-hasura-role``)
```

>        is sent along with admin secret header (``x-hasura-admin-secret``), defaults to ``false``

Reviewer: ```suggestion
       is sent along with the admin secret header (``x-hasura-admin-secret``), default: ``false``
```

     - false
     - :ref:`PGColumn` array (or) ``'*'``
     - Can insert into only these columns (or all when ``'*'`` is specified)
   * - backend_only
     - false
     - Boolean

>      - When set to ``true`` the mutation is accessible only if ``x-hasura-backend-privilege`` session variable exists and set to ``true``

Reviewer: ```suggestion
     - When set to ``true`` the mutation is accessible only if ``x-hasura-backend-privilege`` session variable exists and is set to ``true``
```
